lib/messaging/messagehandler.rb

```diff
@@ -22,7 +28,7 @@ class MessageHandler
 
   def initialize
     #@dbaccess = DbAccess.new
-    @dbaccess = $dbaccess
+    @dbaccess = $dbaccess || NullDatabase.new
```

Comment:
So if I moved this into its own process, I could remove the reference to the database completely?